server/pulp/server/db/model/__init__.py

```diff
@@ -983,6 +983,24 @@ class CeleryBeatLock(AutoRetryDocument):
     _ns = StringField(default='celery_beat_lock')
 
 
+class ResourceManagerLock(AutoRetryDocument):
+    """
+    Single document collection which gives information about the current resource manager lock.
+
+    :ivar name: string representing the resource manager instance name
+    :type name: basestring
+    :ivar lock: A unique key set to "locked" when lock is acquired.
+    :type lock: basestring
+    :ivar _ns: (Deprecated), Contains the name of the collection this model represents
+    :type _ns: mongoengine.StringField
+    """
+    name = StringField(required=True)
+    lock = StringField(required=True, default="locked", unique=True)
+
+    # For backward compatibility
+    _ns = StringField(default='resource_manager_lock')
```

Comment:
I don't think we need this field on new models. I think it's only needed on models that existed before we started using mongoengine. It's worth querying the team just to be sure.